src/main/java/uk/gov/hmcts/reform/prl/enums/YesNoDontKnow.java

```diff
@@ -15,9 +15,7 @@ public enum YesNoDontKnow {
     @JsonProperty("no")
     no("no", "No"),
     @JsonProperty("dontKnow")
-    dontKnow("dontKnow", "Don't know"),
-    @JsonProperty("iDontKnow")
-    iDontKnow("iDontKnow", "I dont know");
```

Comment:
think this one is in use somewhere.

Reply:
this file also using it - CaseDataChildDetailsElementsMapper

Reply:
@AlokDatta This is new one I added earlier, CaseDataChildDetailsElementsMapper using dontKnow not iDontKnow.
`@JsonProperty("iDontKnow")
iDontKnow("iDontKnow", "I dont know");`